tool/TeamCity.Docker/TeamCityKotlinSettingsGenerator.cs

```diff
@@ -867,31 +872,6 @@ private IEnumerable<string> CreateArtifactsDependencies()
             yield return "}";
         }
 
-        /// <summary>
-        /// Create build configuration trigger dependant on finished build. 
-        /// </summary>
-        /// <param name="id">target build ID</param>
-        /// <param name="useEnclosureForId">indicates if enclosure (${...}) should be used for build ID</param>
-        /// <returns></returns>
-        private IEnumerable<string> CreateFinishBuildTrigger(string id, Boolean useEnclosureForId = false) {
```

Comment:
I've implemented  `CreateFinishBuildTrigger(…)` a few months ago, however, I find it convenient to move towards simplification of DSL generation logic. Using multi-line strings instead of C# method would be a more straightforward approach, which would simplify the maintenance.